EntityFramework.Utilities/EntityFramework.Utilities/DatabaseExtensionMethods.cs

```diff
@@ -23,18 +23,14 @@ public static void ForceDelete(this Database db, string name = null)
 				nameParameter.ParameterName = "@Name";
```

Comment:
can't we get rid of nameParameter then, since you're using string interpolation to directly put the database name in the command string?

Reply:
That was a pretty silly oversight! It's fixed.